contracts/lib/constants.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+/// @dev The approximate number of blocks per year that is assumed by the interest rate model
+uint256 constant BLOCKS_PER_YEAR = 10_512_000;
+
+/// @dev Base unit for computations
+uint256 constant BASE = 1e18;
```

Comment:
can we change this variable name to `expScale` to maintain uniformity?

Reply:
Renamed to EXP_SCALE and MANTISSA_ONE. The reason is that "scale" kinda feels like it should be multiplied/divided upon, it doesn't make much sense to add to or subtract anything from "scale" – MANTISSA_ONE seems more natural.

Fixed in commits f650fa2, 2f54c4c, b22a86c